src/olympia/amo/tests/__init__.py

```diff
@@ -269,6 +269,30 @@ def __getattr__(self, name):
         else:
             raise AttributeError
 
+    def generate_api_token(self, user, **payload_overrides):
+        """
+        Creates a jwt token for this user.
+        """
+        from rest_framework_jwt.settings import api_settings
```

Comment:
why not just import from our own settings? Wouldn't that provide better coverage of our configuration?